code/game/gamemodes/objectives/basic/untracked/cause_trouble.dm

```diff
@@ -0,0 +1,6 @@
+/datum/objective/cause_trouble
+	name = "Cause Trouble"
+	explanation_text = "Cause some interesting trouble for security to deal with. This is always marked as complete, so don't fret if you get arrested. It's all about the drama!"
```

Comment:
This is too vague. Where is the line drawn? No killing? "Interesting trouble" could be anything and players will interpret it as such.